assets/admin/package.json

```diff
@@ -3,29 +3,30 @@
     "license": "MIT",
     "repository": "https://github.com/sulu/skeleton.git",
     "scripts": {
-        "preinstall": "node ../../vendor/sulu/sulu/preinstall.js",
+        "preinstall": "npx symlink-dir ../../vendor node_modules/@sulu/vendor",
         "build": "webpack --mode production",
-        "watch": "webpack --mode development -w"
+        "watch": "webpack --mode development -w",
+        "postinstall": "npx symlink-dir ../../vendor node_modules/@sulu/vendor"
```

Comment:
this should be done in  `node ../../vendor/sulu/sulu/postinstall.js` script

Reply:
isnt this the same as in `preinstall`? 🤔

Reply:
Yeah but our current preinstall script is skipt so in the end result it can not be the same script